styles/style.css

```diff
@@ -6,3 +6,116 @@ Green: #00B172
 White: #FFF
 
 */
+body {
+  margin: 0;
+  font-family: "Open Sans", sans-serif;
+}
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px 0;
+  /* position: fixed; */
```

Comment:
If you want to set the header to position fixed, you can add a background to add and adjust its width
```
  background-color: white;
  width: 100%;
```